frontend/public/components/chargeback.tsx

```diff
@@ -317,23 +316,30 @@ class ReportData extends React.Component<ReportDataProps, ReportDataState> {
 
   render() {
     const {obj} = this.props;
-    const phase = _.get(obj, ['status', 'phase']);
-
+    const conditions = _.get(obj, ['status', 'conditions']);
+    const isFinished = _.some(conditions, {reason: 'Finished'});
```

Comment:
Don't you need to check the true/false value of the condition?

Reply:
You would think so, but I'm always getting `status: false`:  (cc @chancez) 
```
conditions:
    - lastTransitionTime: '2019-05-06T13:52:25Z'
      lastUpdateTime: '2019-05-06T13:52:25Z'
      message: >-
        Report has finished reporting. Report has reached the configured
        spec.reportingEnd: 2019-12-30 23:59:59 +0000 UTC
      reason: Finished
      status: 'False'
```